lochness/email/__init__.py

```diff
@@ -1,7 +1,17 @@
 import os
+from pathlib import Path
 import string
 import smtplib
 from email.mime.text import MIMEText
+from jinja2 import Environment, FileSystemLoader
+from datetime import datetime, date, timedelta
+from typing import List
+import getpass
+import socket
+import subprocess
+import pandas as pd
+from pytz import timezone
+tz = timezone('EST')
```

Comment:
Do we know how EDT will be handled then?

Reply:
Are you talking about the data entry tigger? data entry trigger will use timestamp, which will not be affected by timezones. This should be investigated further when we move to production server, and when the `data entry trigger` is configured on it... I'm not sure when this will happen though.